lib/functional-constraints/searchOrCreateKeys.js

```diff
@@ -11,14 +11,15 @@ const validateSearchOrCreateKeys = (definition) => {
   const errors = [];
   const searchKeys = _.keys(definition.searches);
```

Comment:
Thanks for the suggestion. I think I'll keep it as it is because it's similar enough.